### PR TITLE
Refactor: arrange the order of functions in built-in classes for consistency

### PR DESCRIPTION
**Pull request by hachi8833**
I hope this helps us.

1. type definitions
2. init*Object()
3. init*Class()
4. class methods
5. instance methods
6. polymorphic helper functions
7. other helper functions

### CODE REVIEW

File: vm/error.go [hunk 1]
```diff
@@ -35,6 +35,19 @@ const (
 	UnsupportedMethodError = "UnsupportedMethodError"
 )
 
+// Error ...
```
Comment (st0012) [inline vm/error.go]: @hachi8833 Can you also help document Error struct? thanks 😁

File: vm/vm.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ type filename string
 type errorMessage string
 
 var standardLibraries = map[string]func(*VM){
-	"file":              initializeFileClass,
+	"file":              initFileClass,
 	"net/http":          initializeHTTPClass,
```
Comment (st0012) [inline vm/vm.go]: Can you also change these function names?

File: vm/error.go (hunk 1 above)
Comment (hachi8833) [inline vm/error.go]: I'll look into it!

File: vm/vm.go (hunk 2 above)
Comment (hachi8833) [inline vm/vm.go]: I got it!

File: vm/error.go (hunk 1 above)
Comment (hachi8833) [inline vm/error.go]: I'd do this on other PR

Comment (st0012) [inline vm/error.go]: ok, then I'll merge this first.

**Comment by st0012** [comment]
@hachi8833 Thanks a lot!

**Pull request merged**